api/libproject/environments.rb

```diff
@@ -181,6 +181,36 @@ def env_with_defaults(env, config)
         :auth_domain_group_email => "all-of-us-controlled-prod@firecloud.org",
       }
     }
+  }),
+  "vwb-dev-stable" => env_with_defaults("vwb-dev-stable", {
+    :source_cdr_project => "all-of-us-ehr-dev",
+    :source_cdr_wgs_project => "all-of-us-workbench-test",
+    :publisher_account => "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com",
+    :skip_acl => true,
+    :accessTiers => {
+      "registered" => {
+        :dest_cdr_project => "vwb-dev-rapid-kale-4145",
+      },
+      "controlled" => {
+        :dest_cdr_project => "vwb-dev-buoyant-apple-6238",
+        :dest_cdr_bucket => "gs://fc-aou-test-datasets-controlled",
```

Comment:
`dest_cdr_bucket` is not right, but it's just a  placeholder and you will change later?